app/components/point.jsx

```diff
@@ -0,0 +1,181 @@
+// https://github.com/EnCiv/civil-pursuit/issues/23
+
+'use strict'
+import React, { useState } from 'react'
+import cx from 'classnames'
+import { createUseStyles } from 'react-jss'
+
+function Point(props) {
+  const { subject, description, vState, children, styles, className, ...otherProps } = props
+
+  const [isHovered, setIsHovered] = useState(false)
+
+  const classes = useStylesFromThemeFunction()
+
+  const handleMouseEnter = () => {
+    setIsHovered(true)
+  }
+
+  const handleMouseLeave = () => {
+    setIsHovered(false)
+  }
+
+  const childrenWithProps = React.Children.map(children?.props?.children ?? children, child => {
+    return React.cloneElement(child, {
+      isHovered: isHovered,
+      vState: vState,
+    })
+  })
+
+  const borderClass = cx(classes[`${vState}Border`])
+  const subjectClass = cx(classes[`${vState}Subject`])
+  const descriptionClass = cx(classes[`${vState}Description`])
+
+  return (
+    <>
+      <div className={className} style={styles}>
```

Comment:
To clarify, the props should look like this?
  const { subject, description, vState, children, style, className, ...otherProps } = props
  
... and what is the difference between style and className?

Reply:
className will be used by your componnent, style will not (so don't put it in the list).  If the parent component passes style, it will be in otherProps, which will get added to the outer <div> you are returning.   Also, className is a string where style is an object.